apps/docs/generator/index.ts

```diff
@@ -10,6 +10,7 @@ const main = (command: string[], options: any) => {
 }
 
 // Run everything
+/* @ts-ignore - required for Next.js build */
 const argv = minimist(process.argv.slice(2))
```

Comment:
FYI `apps/docs/generator` has its own `tsconfig.json` without `esModuleInterop`, so the above `minimist` import needs to remain as
```ts
import * as minimist from 'minimist'
```

Needed to add this `@ts-ignore` to get the Next.js project to build - it must be ignoring this `tsconfig.json`. Proper solution is to convert `apps/docs/generator` to use `esModuleInterop`, but probably not worth it because we are likely going to remove `generator` soon.